org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/KotlinDefaultArgumentsFilter.java

```diff
@@ -129,6 +128,17 @@ public void match(final MethodNode methodNode,
 				output.ignore(i, i);
 			}
 		}
+
+		private static int maskVar(final String desc, final boolean constructor) {
+			final Type[] argumentTypes = Type.getMethodType(desc)
+					.getArgumentTypes();
+			int slot = 0;
+			for (int i = 0; i < argumentTypes.length
+					- (constructor ? 1 : 2); i++) {
```

Comment:
What do the magic numbers here mean?

Maybe we can enlight future readers of the code with a local variable:

    int descripiveName = constructor ? 1 : 2;

Reply:
@marchof please re-review